ports/esp32/boards/LILYGO_TTGO_LORA32/board.md

```diff
@@ -0,0 +1,3 @@
+The following files are daily firmware for the LILYGO TTGO LoRa32.
+
+Support for v1.0, v1.2, v1.6 and v2.0.
```

Comment:
It might be worth explaining that these versions are for hardware versions of the board (otherwise it could be confused as software version).

Reply:
Thanks for the review Damien, done.